### ISSUE TITLE
Introduce an easier way to handle unique bundle ids across the team

### ISSUE THREAD

**Issue by azasypkin** [project-lighthouse/camera-app.ios#9]
Currently one should manually update bundle id locally to be able to push code to device. This changes project file and is pretty annoying when you try to commit your changes.

I found two ways:

1. http://stackoverflow.com/questions/38813674/xcode-sharing-the-same-bundle-id-among-multiple-people

2. http://stackoverflow.com/questions/12113551/code-signing-with-multiple-developers

Will see which one is better.

**Comment by azasypkin** [comment]
Ugh, both ways are quite messy, looking at https://codesigning.guide/ right now.

**Comment by azasypkin** [comment]
Will take a break here. So far recommended way is to create team-specific Apple Id and use it to sign the code. + optionally use https://github.com/fastlane/fastlane/tree/master/match#usage

**Comment by Yoric** [comment]
According to @sleroux, there is probably no good way that will let us accept anonymous contributors.

**Comment by azasypkin** [comment]
I still want to check if we can benefit from [fastlane](https://github.com/fastlane/fastlane), but I admit I haven't dig into it yet.

**Comment by Yoric** [comment]
Sure.

Also, @sleroux mentioned on irc that

> you can override project settings using a .xcconfig file that might work for developer specific settings

If we can do that, we can generate a unique bundle id in `bootrap.sh`.

**Comment by azasypkin** [comment]
Yep, that's probably the way №1 mentioned in https://github.com/project-lighthouse/camera-app.ios/issues/9#issue-200361920 and should do the job.
